be/src/agent/heartbeat_server.cpp

```diff
@@ -104,6 +106,10 @@ void HeartbeatServer::heartbeat(THeartbeatResult& heartbeat_result, const TMaste
 #endif
         heartbeat_result.backend_info.__set_version(get_short_version());
         heartbeat_result.backend_info.__set_num_hardware_cores(num_hardware_cores);
+        heartbeat_result.backend_info.__set_is_first_heartbeat(first_heartbeat);
```

Comment:
What happens if fe change mater?

Reply:
isFirstHeartbeat is a property in be, fe change leader has no effect on this pr’s logic

Reply:
use reboot time to instead first_heartbeat